CONTRIBUTING.md

```
@@ -51,4 +51,45 @@ logger:
 
 ## Use a Consistent Coding Style
 
-Use [black](https://github.com/psf/black) or [Ruff](https://docs.astral.sh/ruff/) to make sure the code follows the style.
+Use [ruff](https://docs.astral.sh/ruff/) to make sure the code follows the style.
+
+## Setting up the Development Environment
+
+1. Create a virtual environment:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)